tokio-util/src/task/mod.rs

```diff
@@ -11,3 +11,5 @@ pub use join_map::{JoinMap, JoinMapKeys};
 
 pub mod task_tracker;
 pub use task_tracker::TaskTracker;
+
+pub mod spawn_abortable;
```

Comment:
I'm not sure this module name makes much sense considering the type is called `AbortOnDropHandle`.

Reply:
renamed `abort_on_drop.rs`